src/components/PageSettingsModal/PageSettingsModal.tsx

```diff
@@ -146,6 +147,23 @@ export const PageSettingsModal = ({
     })
   }
 
+  const getDefaultPermalink = (title: string) => {
+    if (!title) return ""
+    const titleSlug = title
+      .toLowerCase()
+      .replace(/[^a-z0-9]+/g, "-")
```

Comment:
should we replace `A-Z`? maybe the regex should include `A-Z` to prevent replacement

Reply:
hey, i intentionally wanted to replace. we did a similar thing regarding the amplify migration, where we changed all the files to lowercase in an attempt to standardise things. my hunch here is that making all permalinks lowercase helps us standardise things, now that we are using a infra(amplify) that is case sensitive

Reply:
oops, i didn't see the `toLowerCase`